packages/dialect-react/tsconfig.json

```diff
@@ -1,34 +1,13 @@
 {
+  "extends": "../../tsconfig.cjs.json",
   "compilerOptions": {
-    "target": "es2019",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
-    "declaration": true,
-    "outDir": "lib/cjs",
-    "allowJs": true,
-    "skipLibCheck": true,
-    "strict": true,
-    "forceConsistentCasingInFileNames": true,
-    "esModuleInterop": true,
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "jsx": "react",
-    "sourceMap": true,
-    "noEmit": false,
-    "importsNotUsedAsValues": "error"
+    "outDir": "lib/cjs/"
   },
   "include": [
     "**/*.ts",
     "**/*.tsx"
   ],
   "exclude": [
-    "node_modules",
-    "lib"
-  ],
-  "type": "module"
+    "node_modules"
```

Comment:
I would still exclude lib folder, to be not included in the compilation, unless there is a specific reason why you've removed it.

Reply:
good point. I don't think that we'll have ts files in lib, but yeah, why not